app/build.gradle

```diff
@@ -82,6 +82,7 @@ dependencies {
 
     implementation 'org.hamcrest:hamcrest:2.2'
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+    implementation 'androidx.wear:wear:1.0.0'
```

Comment:
Are those really needed, I can't find any usage of this in our project and we are not supporting smart watches, event if that could be fun ;)

Reply:
Oh no it came back! 
I created a BlankActivity at first for the Edits instead of an EmptyActivity and apparently they're used in apps for watches x)
(I thought I removed them on this branch but must have done so in another...)